cob/cob.h

```diff
@@ -267,11 +267,22 @@ struct m0_cob_domain {
 	 * m0_be_btree has it's own volatile-only fields, so it can't be placed
 	 * before the m0_format_footer, where only persistent fields allowed
 	 */
-	struct m0_be_btree      cd_object_index;
-	struct m0_be_btree      cd_namespace;
-	struct m0_be_btree      cd_fileattr_basic;
-	struct m0_be_btree      cd_fileattr_omg;
-	struct m0_be_btree      cd_fileattr_ea;
+	struct m0_btree *cd_object_index;
+	struct m0_btree *cd_namespace;
+	struct m0_btree *cd_fileattr_basic;
+	struct m0_btree *cd_fileattr_omg;
+	struct m0_btree *cd_fileattr_ea;
+	uint8_t          cd_oi_node[1024] __attribute__((aligned(1024)));
+	uint8_t          cd_ns_node[1024] __attribute__((aligned(1024)));
+	uint8_t          cd_fa_basic_node[1024] __attribute__((aligned(1024)));
+	uint8_t          cd_fa_omg_node[1024] __attribute__((aligned(1024)));
+	uint8_t          cd_fa_ea_node[1024] __attribute__((aligned(1024)));
```

Comment:
The max value size of `ea` btree is more than 4096 as per below code. 
```
 241 M0_INTERNAL size_t m0_cob_max_earec_size(void)
 242 {
 243         return sizeof(struct m0_cob_earec) + M0_COB_EA_MAX;
 244 }

  67 enum {
  68         M0_COB_NAME_MAX = 256,
  69         M0_COB_EA_MAX   = 4096
  70 };
```
The max value size + key size will be greater than 1024. Need to allocate bigger node for `ea` type tree. Please check size for `namespace` tree too.

Reply:
ok.